docs/DeveloperGuide.md

```diff
@@ -312,7 +312,16 @@ These requirements can be restrictive for agents who may prefer more flexible fi
 
 **Status:** Scheduled for future release due to current timeline priorities.
 
-### 5. Warning for Edit Commands with Inputs Identical to Current Values
+### 7. Prevent Duplicate Client Entries by Standardizing Name Formatting
+**Current issue:** Client names are currently considered distinct if they contain minor differences in whitespace (e.g., `John Doe` vs. `John  Doe`). This can lead to duplicate entries due to minor typos when using the `add` or `edit` commands.
+
+This can result in duplication of clients when using the `add` or `edit` command due to typos.
```

Comment:
```suggestion
**Current issue:** Duplicate client entries can occur when minor differences in whitespace are present in names due to user input errors (e.g., `John Doe` vs. `John   Doe`). These variations, often caused by unintentional typos when using the `add` or `edit` commands, are currently treated as distinct entries.
```
```

Reply:
Just an update to highlight that it is caused by user input error rather than strictly incorrect behaviour.